editor/test/integration/test_util.clj

```diff
@@ -696,6 +725,7 @@
          resource-type (resource/resource-type resource)
          write-fn (:write-fn resource-type)
          content (write-fn pb-map)]
+     (fs/create-parent-directories! (io/as-file resource))
```

Comment:
Make sure parent directories exist for these to not get errors when creating stuff below a path featuring non-existing directories.